src/trials.jl

```diff
@@ -313,36 +340,139 @@ Base.show(io::IO, t::TrialJudgement) = _show(io, t)
+    rmaxtimewidth = maximum(length.([stdtime, maxtime]))
+    lmaxgcwidth = maximum(length.([medgc, avggc, mingc]))
+    rmaxgcwidth = maximum(length.([stdgc, maxgc]))
+
+    # Main stats
+
     pad = get(io, :pad, "")
-    println(io, pad, "  memory estimate:  ", memorystr)
-    println(io, pad, "  allocs estimate:  ", allocsstr)
-    println(io, pad, "  --------------")
-    println(io, pad, "  minimum time:     ", minstr)
-    println(io, pad, "  median time:      ", medstr)
-    println(io, pad, "  mean time:        ", meanstr)
-    println(io, pad, "  maximum time:     ", maxstr)
-    println(io, pad, "  --------------")
-    println(io, pad, "  samples:          ", length(t))
-    print(io,   pad, "  evals/sample:     ", t.params.evals)
+    print(io, "BechmarkTools.Trial: ")
+    printstyled(io, length(t); bold=true)
+    print(io, " samples with ")
+    printstyled(io, t.params.evals; bold=true)
+    print(io, " evaluations.")
+
+    print(io, "\n", pad, " Time  ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "median"; color=:blue, bold=true)
+    printstyled(io, "):     "; color=:light_black)
+    printstyled(io, lpad(medtime, lmaxtimewidth), rpad(" ", rmaxtimewidth + 5); color=:blue, bold=true)
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "median")
+    printstyled(io, "):    "; color=:light_black)
+    print(io, lpad(medgc, lmaxgcwidth))
+
+    print(io, "\n", pad, " Time  ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "mean"; color=:green, bold=true)
+    print(io, " ± ")
+    printstyled(io, "σ"; color=:green)
+    printstyled(io, "):   "; color=:light_black)
+    printstyled(io, lpad(avgtime, lmaxtimewidth); color=:green, bold=true)
+    print(io, " ± ")
+    printstyled(io, lpad(stdtime, rmaxtimewidth); color=:green)
+    print(io, "  ")
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "mean ± σ")
+    printstyled(io, "):  "; color=:light_black)
+    print(io, lpad(avggc, lmaxgcwidth), " ± ", lpad(stdgc, rmaxgcwidth))
+
+    print(io, "\n", pad, " Range ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "min"; color=:cyan, bold=true)
+    print(io, " … ")
+    printstyled(io, "max"; color=:magenta)
+    printstyled(io, "):  "; color=:light_black)
+    printstyled(io, lpad(mintime, lmaxtimewidth); color=:cyan, bold=true)
+    print(io, " … ")
+    printstyled(io, lpad(maxtime, rmaxtimewidth); color=:magenta)
+    print(io, "  ")
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "min … max")
+    printstyled(io, "): "; color=:light_black)
+    print(io, lpad(mingc, lmaxgcwidth), " … ", lpad(maxgc, rmaxgcwidth))
+
+    # Histogram
+
+    histtimes = t.times[1:round(Int, 0.99*end)]
```

Comment:
What is the goal here? Is this supposed to be `t.times[1:end-1]`?

Reply:
I found that without slicing off the top 1% of times anomalous results would dramatically increase the maximum end of the scale, squashing 99% of the data onto 10% of the scale (or similar). I think this is pretty undesirable.

Extracted to a helpfully named variable and annotation added to the scale for less "magic number"-ness.

Reply:
in BenchmarkHistograms I resolved this by using the 0.999 quantile of the data

> We choose a value Q = quantile(x, 0.999) to represent close-to-the-upper end of the data we might expect. Then, if Q is more than 2 bins away from the actual maximum, we collapse all of those bins into a single bin.

https://github.com/ericphanson/BenchmarkHistograms.jl/pull/13